docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -25,22 +25,50 @@ adding a new item to the list of options or managing a selection.
 
 Combobox is composed of three main parts:
 
-#### Trigger
+#### Activator
 
-The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
+The activator is the user-facing interface to open and close the Combobox and
+initiate option selection. When implementing Combobox, you may rely on the
+default activator or, you may pass in a custom activator.
 
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+##### Default activator
+
+The default activator is a
```

Comment:
This is totally fair however putting `Combobox` inside a Canvas tag and getting it to render is surprisingly difficult, at least for me right now. I can try with a different set of eyes tomorrow.  
That being said, we were also trying to steer clear of too many live examples within the docs page (because otherwise why have examples as well).  This is getting into a larger conversation about what our docs should look like and we have yet to return to that conversation (me + you + survey results).
So if I can get a Canvas in there relatively quickly tomorrow I will - if not I'll put this into the backlog because I do agree, having a live example here makes sense.